openchemistry/api.py

```diff
@@ -88,13 +88,12 @@ def _calculation_monitor(taskflow_ids):
 
     return table
 
-def import_structure(smiles=None, inchi=None, cjson=None, gen3d=True):
+def import_structure(smiles=None, inchi=None, cjson=None, gen3d=True, params={}):
```

Comment:
`params` has a mutable default value. The value will be assigned when the function is defined, so the same value will be used for each invocation that doesn't provide `params`, which is probably not intended as we are writing to it in the function. We should default it to `None` and then provide a new `dict` inside the function.

Reply:
Thank you for pointing this out! Fixed